### DISCUSSION TITLE
Markdown Toolbar: Add buttons to indent / outdent

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#688] [Feature Requests]
Currently this can be done by long pressing the `<` and `>` buttons. But that's not very discoverable. A user has requested actual buttons for it.

I'm a bit conflicted since the toolbar is already quite full.

**Comment by gadabout84** [comment]
@vHanda, the long pressing feature doesn't work on iOS though.
Should this already work, or isn't it implemented for iOS yet?

In addition the markdown-editor menu on an iPhone X is cropped to the right:
![image](https://user-images.githubusercontent.com/5605669/180773574-23036bd8-64a3-433d-a02c-94365ec18576.png)

**Reply by bixycler** [reply]
Neither on Android app!

**Comment by bixycler** [comment]
Yeah! I definitely begging for this feature!  The `>`, `<` buttons for one-word jump is not as necessary as the indent & outdent function, provided the lack of Tab key in mobile keyboards.
